README.md

```diff
@@ -1,8 +1,194 @@
+  * [Automatic tests](#automatic-tests)
+  * [Test creation](#test-creation)
+  * [Testing on real hardware](#testing-on-real-hardware)
+  * [Ethercat interface](#ethercat-interface)
+- [Deployment](#deployment)
+- [Structure of files](#structure-of-files)
+  * [Roles](#roles)
+  * [Docker](#docker)
+  * [Installation](#installation)
+  * [Products](#products)
+  * [Common](#common)
+  * [Dependencies](#dependencies)
+- [Playbooks and possible command line arguments](#playbooks-and-possible-command-line-arguments)
+  * [Playbook creation](#playbook-creation)
+  * [teleop_deploy](#teleop_deploy)
+  * [docker_deploy](#docker_deploy)
+  * [configure_software](#configure_software)
+  * [install_software](#install_software)
+  * [install_python3](#install_python3)
+- [Inventories](#inventories)
+- [Molecule tests](#molecule-tests)
+  * [Docker tests](#docker-tests)
+  * [AWS EC2 tests](#aws-ec2-tests)
+- [Syntax and rules](#syntax-and-rules)
+- [Tutorial 1 desktop icon](#tutorial-1-desktop-icon)
 
-The main purpose of this subsystem is to unify one-liners approaches based on Ansible best practices. This one-liner is able to install Docker, download the specified image and create a new container for you. It will also create a desktop icon to start the container and launch the hand.
+# Introduction #
 
-## Before running the docker_deploy playbook ##
+Aurora is an installation automation tool using Ansible. It uses Molecule for testing Ansible scripts and it has automated builds in AWS EC2/CodeBuild and DockerHub. It can be used to develop, test and deploy complicated, multi-machine, multi-operating-system automated installs of software. Aurora's purpose is to unify one-liners approaches based on Ansible best practices.
+
+For example, it's possible to use Aurora to install Docker, download the specified image and create a new container for you. It will also create a desktop icon to start the container and launch the hand.
+
+Ansible user guide is available [here](https://docs.ansible.com/ansible/latest/user_guide/index.html) (Aurora is currently using Ansible 2.8.1)
+
+Molecule user guide is available [here](https://molecule.readthedocs.io/en/stable/) (Aurora is currently using Molecule 2.20.1)
+
+# Development #
+
+The recommended way to develop code for this project is to pull a certain docker image ([Development Docker](#development-docker)) with a lot of tools already installed and open a container of this image, then clone the aurora GitHub repository inside it. It is not recommended to clone aurora directly on your local machine while you do development and testing.
+
+## Development Docker ##
+
+The docker images used for aurora development are [here](https://cloud.docker.com/u/shadowrobot/repository/docker/shadowrobot/aurora-molecule-devel)
+
+Currently both xenial and bionic tags are working well and there is no difference.
+The rest of the document assumes the user is using bionic tag.
+
+Instructions on how to use this:
```

Comment:
maybe we can have a aurora-deploy playbook later to automate this?

Reply:
Yes, a good idea. Can you please create a small user story?